lib/src/fhir_rest_client.dart

```diff
@@ -9,6 +9,32 @@ class FhirRestClient {
 
   FhirRestClient({required this.dio, required this.baseUrl});
 
+  Future<List<String>> getSchema() async {
+    final response = await dio.get(
+      baseUrl.replace(path: 'metadata').toString(),
+      options: Options(
+        headers: {
+          'Accept': 'application/fhir+json',
+          'Content-type': 'application/fhir+json',
+        },
+      ),
+    );
+
+    final data = response.data as Map<String, dynamic>;
+    final resources = data['rest'][0]['resource'] as List<dynamic>;
+
+    final schema = resources
```

Comment:
line 26-33 should be upstream in the caller

Reply:
563f058